### DISCUSSION TITLE
Prevent Horizontal Scrolling on Mobile

### DISCUSSION THREAD

**Discussion by westocl** [jothepro/doxygen-awesome-css#66] [Ideas]
Thanks for creating this project, the mobile view for Doxygen is 1000% better than the original!

I do have one question though. Is there a way to stop the horizontal scrolling with CSS on mobile phones?  Everything is perfect on the desktop, but on my Iphone, when you move your thumb it makes the content scroll left and right. 

Not sure if you have seen this on your mobile.  I have tried to add:

body {
   height: 100%;
   overflow-y: hidden;
}

But that doesn't seem to work.  Any suggestions?

**Comment by jothepro** [comment]
Horizontal scrolling on small screens because some page content is too wide and refuses to scale down is a problem that I've struggled with a lot lately. What you are describing may be a bug, my ultimate goal is to eliminate this kind of behavior.

Can you share a screenshot/screencast or a link to a page where the scrolling occurs, so I can reproduce it? What version of Doxygen and doxygen-awesome-css are you using? Does this also occur to you somewhere [on the demo page](https://jothepro.github.io/doxygen-awesome-css/)?

**Comment by westocl** [comment]
Hey thanks for the reply, the site is not fully up yet, so not sure how to get it so you can see it with your phone.

If you know the IP address of how you will be accessing the site I can put that IP address as an allow, so you can see it but it will be still blocked under construction to the public
